src/decoder/plugins/SidplayDecoderPlugin.cxx

```diff
@@ -65,6 +66,38 @@ static unsigned default_songlength;
 
 static bool filter_setting;
 
+#ifdef HAVE_SIDPLAYFP
+static const unsigned romSize = 8192;
+static std::unique_ptr<uint8_t[]> kernal(new uint8_t[romSize]());
+static std::unique_ptr<uint8_t[]> basic(new uint8_t[romSize]());
```

Comment:
I don't like that these buffers get allocated even if no ROM path is configured. Also why call the constructor on the allocated array?

Reply:
Come on, its only 16 kb ;) - Please allow some time for making requested changes.